File: ClinicSystem/src/com/example/clinicsystem/models/classes/M_SterilizationTool.java
package com.example.clinicsystem.models.classes;

import com.example.clinicsystem.helpers.DatabaseConnection;
import com.example.clinicsystem.models.enums.EquipmentType;

public class M_SterilizationTool extends M_Equipment {
    private String sterilizationType;

    public M_SterilizationTool(String name, EquipmentType equipmentType, int price, int id, int quantity, boolean isAvailable, String sterilizationType) {
        super(name, equipmentType, price, id, quantity, isAvailable);
        this.sterilizationType = sterilizationType;
    }

    public M_SterilizationTool() {
    }

    public int addSterilization() {
        DatabaseConnection databaseConnection = DatabaseConnection.getINSTANCE();

        return databaseConnection.insertInto("sterilization_tool", "(Equipment_ID, Sterilization_Type)",
                "('" + this.getId() + "', '" + this.sterilizationType + "')");
    }

    public String getSterilizationType() {
        return sterilizationType;
    }

    public void setSterilizationType(String sterilizationType) {
        this.sterilizationType = sterilizationType;
    }
}

File: clinic/M_SterilizationTool.php
<?php
include_once "M_Equipment.php";
class M_SterilizationTool extends M_Equipment {
    private $sterilizationType;
	public $myID;

    public function __construct($name, $equipmentType, $price, $id, $quantity, $isAvailable, $sterilizationType) {
		$this->name =          $name;
        $this->equipmentType = $equipmentType;
        $this->price =         $price;
        $this->id =            $id;
        $this->quantity =      $quantity;
        $this->isAvailable =   $isAvailable;
        $this->sterilizationType = $sterilizationType;
    }

    public function addSterilization() {
        $conn = DatabaseConnection::getINSTANCE();
		
		return $conn->insertInto("sterilization_tool", "(Equipment_ID, Sterilization_type)",
			"('". $this->myID . "', '" . $this->sterilizationType . "')");
    }

    public function getSterilizationType() {
        return $this->sterilizationType;
    }

    public function setSterilizationType($sterilizationType) {
        $this->sterilizationType = $sterilizationType;
    }
}
?>